bridge_test.go

```diff
@@ -422,6 +422,36 @@ func TestHandlePacket(t *testing.T) {
 					CLabels:     map[string]string{},
 				},
 			},
+		}, {
+			name: "ms timer with conversion to seconds",
+			in:   "foo:200|ms",
+			out: event.Events{
+				&event.TimerEvent{
+					TMetricName: "foo",
+					TValue:      0.2,
+					TLabels:     map[string]string{},
+				},
+			},
+		}, {
+			name: "histogram with no unit conversion",
+			in:   "foo:200|h",
+			out: event.Events{
+				&event.TimerEvent{
+					TMetricName: "foo",
+					TValue:      200,
+					TLabels:     map[string]string{},
+				},
+			},
+		}, {
+			name: "distribution with no unit conversion",
+			in:   "foo:200|d",
+			out: event.Events{
+				&event.TimerEvent{
+					TMetricName: "foo",
+					TValue:      200, // convert statsd ms to seconds in parsing
```

Comment:
I think this comment is out of date now
```suggestion
					TValue:      200,
```

Reply:
Yeah, sorry forgot to remove that when I was fixing benchmarks